pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -67,6 +69,22 @@
                       'the returned class(es). '
                       'Default: Include superclass properties and methods.')]
 
+associations_option = [              # pylint: disable=invalid-name
+    click.option('--association/--no-association',
+                 default=None,
+                 help='Returns only that are either association classes '
+                      '(--association) or are not association classes'
+                      '(--not-association). If the option is not defined '
+                      'no filtering occurs')]
+
+indications_option = [              # pylint: disable=invalid-name
+    click.option('--indication/--no-indication',
+                 default=None,
+                 help='Returns only that are either indication classes '
+                      '(--indication) or are not indication classes'
+                      '(--not-indication). If the option is not defined '
```

Comment:
Fix the name of the negative option in the help text: `--not-indication` -> `--no-indication`.

Reply:
DONE